### PR TITLE
Update digital twin samples to support net core 3.0

### PR DESCRIPTION
**Pull request by timtay-microsoft**
Previously only supported net core 2.2, but now supports both 2.2 and 3.0

### CODE REVIEW

File: digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs [hunk 1]
```diff
@@ -38,11 +49,11 @@ protected override async Task<DigitalTwinCommandResponse> OnCommandRequest(Digit
                 string commandPayload = JsonConvert.DeserializeObject<string>(commandRequest.Payload);
                 if (commandPayload.Equals(EnvironmentalSensorInterface.EnvironmentalSensorInterfaceId))
                 {
-                    return new DigitalTwinCommandResponse(StatusCodeCompleted, this.environmentalSensorModelDefinition);
+                    return Task<DigitalTwinCommandResponse>.Factory.StartNew(() => new DigitalTwinCommandResponse(StatusCodeCompleted, this.environmentalSensorModelDefinition));
```
Comment (drwill-ms) [inline digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs]: This should just be:

return Task.FromResult(new DigitalTwinCommandResponse(StatusCodeCompleted, this.environmentalSensorModelDefinition));

File: digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs [hunk 2]
```diff
@@ -18,18 +19,28 @@ public class ModelDefinitionInterface : DigitalTwinInterfaceClient
...
+        private const string EnvironmentalSensorInterfaceFileAssemblyName = "EnvironmentalSensorSample";
+
         public ModelDefinitionInterface(string interfaceInstanceName)
             : base(ModelDefinitionInterfaceId, interfaceInstanceName)
         {
-            environmentalSensorModelDefinition = File.ReadAllText("../../../EnvironmentalSensor.interface.json");
+            var assembly = Assembly.GetExecutingAssembly();
+            var assemblyResourceName = EnvironmentalSensorInterfaceFileAssemblyName + "." + EnvironmentalSensorInterfaceFileName;
+
+            using (Stream stream = assembly.GetManifestResourceStream(assemblyResourceName))
```
Comment (drwill-ms) [inline digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs]: Consider using the new syntax:

using Stream stream = assembly.GetManifestResourceStream(assemblyResourceName);
using StreamReader reader = new StreamReader(stream);

No need for code blocks anymore. When the variables go out of scope, Dispose will be called.

Comment (timtay-microsoft) [reply digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs]: Cool, I didn't know that was an option!

File: digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs (hunk 1 above)
Comment (timtay-microsoft) [reply digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs]: Sure thing

**Pull request merged**